src/Components/Grants/SortableItem.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { useSortable } from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import { Item } from "./Item";
+import countSectionWords from "../../Helpers/countSectionWords";
+import "./SortableItem.css";
+import { MdDragIndicator } from "react-icons/md";
+
+export function SortableItem(props) {
```

Comment:
wyt of making this more specific and calling it `SortableSection`?

Reply:
This is a react-dnd syntax thing - let's leave it for now and circle back when we've finished implementing the feature.

Reply:
We choose this naming because if follows the naming logic in the dnd-kit documentation.